packages/component-card--product/dist/card--product.css

```diff
@@ -98,171 +98,19 @@
   --color-red-error: #d63118;
   --color-red-error-light: #fef3f4;
 }
-.coop-c-editorialcard {
```

Comment:
Please confirm the removal of these styles was intentional.

Reply:
@jcpeden Yes the product card does not reply on this file anymore.